packages/component-base/test/element-mixin.test.js

```diff
@@ -35,6 +36,27 @@ describe('ElementMixin', () => {
     });
   });
 
+  describe('cancelSyntheticClickEvents', () => {
+    let button;
+
+    beforeEach(() => {
+      button = document.createElement('div');
```

Comment:
This looks confusing: should it be `button` element and not `div`? 
Otherwise the variable can be probably renamed.

Reply:
Good point, fixed.